styles.css

```diff
@@ -68,7 +68,7 @@ body{
   }
   
   input[type=checkbox]{
-    border: 0px;
+    border: 0;
```

Comment:
Good job implementing the border but i will highly suggest that you remove one border from the element because it will lead to linters error.